web/src/components/ErrorBoundary.tsx

```
@@ -15,21 +17,76 @@ class ErrorBoundary extends Component<ErrorBoundaryProps, ErrorBoundaryState> {
+                shadow-neu-flat hover:text-accent
+                active:shadow-neu-pressed active:scale-[0.98]
+                transition-all duration-200
+              "
+            >
+              <FiRefreshCw className="text-lg group-hover:rotate-180 transition-transform duration-500" />
+              <span>Reboot System</span>
+            </button>
+          </div>
+
+          <div className="absolute bottom-8 text-[10px] text-text-secondary/30 font-mono uppercase tracking-[0.3em]">
+            NYX Secure Protocol // Critical Failure
```

Biome (2.4.6)

[error] Wrap comments inside children within braces.

(lint/suspicious/noCommentText)